pkg/pdutil/clock.go

```diff
@@ -100,10 +109,10 @@ func (c *clock) Run(ctx context.Context) {
 // CurrentTime returns current time from timeCache
 func (c *clock) CurrentTime() (time.Time, error) {
 	c.mu.RLock()
-	err := c.mu.err
-	cacheTime := c.mu.timeCache
-	c.mu.RUnlock()
-	return cacheTime, errors.Trace(err)
+	defer c.mu.RUnlock()
+	tsEventTime := c.mu.tsEventTime
+	current := tsEventTime.Add(time.Since(c.mu.tsProcessingTime))
```

Comment:
why can we just return `tsEventTime.Add(time.Since(c.mu.tsProcessingTime))`?

Reply:
To make code easier to understand